### PR TITLE
Added options to disable LA setting the terminal title

### PR DESCRIPTION
**Pull request by KarlOfDuty**
Also fixed the line endings setting in the .editorconfig stating `crlf` while all the files in this repo have `lf` line endings.

Also fixed the config wizard not asking about the heartbeat monitoring setting.

### CODE REVIEW

**Comment by ALEXWARELLC** [comment]
This seems ok, but you'll have to wait for @zabszk to review this.

**Comment by KarlOfDuty** [comment]
> EDIT: Merge the pull request i've made to update the README with the updated table.

no u

**Comment by ALEXWARELLC** [comment]
> > EDIT: Merge the pull request i've made to update the README with the updated table.
> 
> 
> 
> no u

Damn it

**Comment by zabszk** [comment]
Hi,

What's the use case scenario for disabling changing the title?

**Comment by KarlOfDuty** [comment]
It stops the title from changing

**Comment by zabszk** [comment]
> It stops the title from changing

I know, but why would someone want to disable that? Does changing the title break something or cause some bugs? You added it both as an arg, in the config and to the config wizard, which is designed for features that many users might want to change.

**Comment by KarlOfDuty** [comment]
image

The title is very long and I can't fit all my server tabs on the screen as compared to the MA titles on the right.

**Comment by zabszk** [comment]
> The title is very long and I can't fit all my server tabs on the screen as compared to the MA titles on the right.

Ok. That's a good reason.

We are still internally discussing the CRLF/LF config. On the repo it's LF, because git normalizes it to LF and then normalizes it back (on Windows) on checkout.

**Comment by KarlOfDuty** [comment]
Normal git doesn't do that, I assume you are talking about the modded cygwin version of git which has settings like that. But that is just an option that you can turn off so it goes back to normal.

The issue is that if you use normal git for this you get a diff where the entire file has changed just because the editorconfig causes your editor to change all lines.

**Comment by ALEXWARELLC** [comment]
> Normal git doesn't do that, I assume you are talking about the modded cygwin version of git which has settings like that. But that is just an option that you can turn off so it goes back to normal. Edit: Well normal git DOES have options for that too, but I assume you are talking about the cygwin specific one that is on by default.
> 
> The issue is that if you use normal git for this you get a diff where the entire file has changed just because the editorconfig causes your editor to change all lines.

I wish there was an option to disable this.

**Comment by KarlOfDuty** [comment]
There is, it asks you about it when you install it.

**Comment by ALEXWARELLC** [comment]
> There is, it asks you about it when you install it.

Well, I'm the dumb one.

File: Core/TcpServer.cs [hunk 1]
```diff
@@ -139,11 +139,11 @@ public void Start()
                                     break;
 
                                 case OutputCodes.IdleEnter:
-                                    Console.Title = "[IDLE] " + LocalAdmin.BaseWindowTitle;
+                                    LocalAdmin.SetTerminalTitle("[IDLE] " + LocalAdmin.BaseWindowTitle);
```
Comment (ALEXWARELLC) [inline Core/TcpServer.cs:142]: You could use interpolation for this if you want. It's just personal preference.

**Review by ALEXWARELLC — COMMENTED** [review]
This is just personal preference but because the BaseWindowTitle is already set, you could use interpolation instead, I don't believe this would change the resulting IL Code though. It's really up to scalability. But this is alright because it still works and works well.

**Comment by ALEXWARELLC** [comment]
Hey, @KarlOfDuty 

Just letting you know that this PR might contain conflicts now #77 has been merged.

**Comment by ALEXWARELLC** [comment]
Everything has been resolved.  Now we wait.

**Review by zabszk — APPROVED** [review]

**Pull request merged**